### PR TITLE
Added licenses and dependencies for chunk1

### PR DESCRIPTION
**Pull request by jsvd**

### CODE REVIEW

File: tools/dependencies-report/src/main/resources/licenseMapping.csv [hunk 1]
```diff
@@ -42,3 +42,40 @@ dependency,dependencyUrl,licenseOverride
 "org.eclipse.jdt:org.eclipse.jdt.core:3.10.0",z,EPL-1.0
 "org.eclipse.osgi:org.eclipse.osgi:3.7.1",z,EPL-1.0
 "org.eclipse.text:org.eclipse.text:3.5.101",z,EPL-1.0
+
+"addressable:2.3.8",https://github.com/sporkmonger/addressable,Apache-2.0
+"asm:6.0",http://asm.objectweb.org,BSD-3-Clause
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/licenseMapping.csv]: No notice file

File: tools/dependencies-report/src/main/resources/licenseMapping.csv [hunk 2]
```diff
@@ -42,3 +42,40 @@ dependency,dependencyUrl,licenseOverride
...
+"avl_tree,1.2.1",https://github.com/nahi/avl_tree,BSD-2-Clause-FreeBSD
+"awesome_print:1.8.0",https://github.com/awesome-print/awesome_print,MIT
+"aws-sdk-core:2.11.37",http://github.com/aws/aws-sdk-ruby,Apache-2.0
+"aws-sdk-resources:2.11.37",http://github.com/aws/aws-sdk-ruby,Apache-2.0
+"aws-sdk-v1:1.67.0",http://aws.amazon.com/sdkforruby,Apache-2.0
+"aws-sdk:2.11.37",http://github.com/aws/aws-sdk-ruby,Apache-2.0
+"aws-sigv4:1.0.2",http://github.com/aws/aws-sdk-ruby,Apache-2.0
+"backports:3.11.3",http://github.com/marcandre/backports,MIT
+"bindata:2.4.3",http://github.com/dmendel/bindata,BSD-2-Clause
+"bouncycastle:",https://www.bouncycastle.org/license.html,MIT
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/licenseMapping.csv]: No notice file

File: tools/dependencies-report/src/main/resources/licenseMapping.csv [hunk 3]
```diff
@@ -42,3 +42,40 @@ dependency,dependencyUrl,licenseOverride
...
+"buftok:0.2.0",https://github.com/sferik/buftok, MIT OR Ruby|BSD-2-Clause
+"builder:3.2.3",http://onestepback.org,MIT
+"bundler:1.9.10",https://bundler.io/,MIT
+"bytelist:",http://github.com/jruby/bytelist,EPL-1.0
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/licenseMapping.csv]: No notice file

File: tools/dependencies-report/src/main/resources/licenseMapping.csv [hunk 4]
```diff
@@ -42,3 +42,40 @@ dependency,dependencyUrl,licenseOverride
...
+"coderay:1.1.2",http://coderay.rubychan.de,MIT
+"com.fasterxml.jackson.core:jackson-annotations:2.9.5",https://github.com/FasterXML/jackson-annotations,Apache-2.0
+"com.fasterxml.jackson.core:jackson-core:2.9.5",https://github.com/FasterXML/jackson-core,Apache-2.0
+"com.fasterxml.jackson.core:jackson-databind:2.9.5",https://github.com/FasterXML/jackson-databind,Apache-2.0
+"com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.9.5",https://github.com/FasterXML/jackson-dataformats-binary,Apache-2.0
+"com.fasterxml.jackson.module:jackson-module-afterburner:2.9.5",https://github.com/FasterXML/jackson-modules-base,Apache-2.0
+"com.google.googlejavaformat:google-java-format:1.1",https://github.com/google/google-java-format,Apache-2.0
+"com.google.guava:guava:19.0",https://github.com/google/guava,Apache-2.0
+"commons-codec:commons-codec:1.10.0",http://commons.apache.org/proper/commons-codec/,Apache-2.0
+"commons-logging:commons-logging:1.2.0",http://commons.apache.org/proper/commons-logging/,Apache-2.0
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/licenseMapping.csv]: No notice files for commons libraries.

File: tools/dependencies-report/src/main/resources/licenseMapping.csv [hunk 5]
```diff
@@ -42,3 +42,40 @@ dependency,dependencyUrl,licenseOverride
...
+"concurrent-ruby:1.0.5",http://www.concurrent-ruby.com,MIT
+"controls.js:",https://github.com/controlsjs/controls.js,GPL-3.0-only
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/licenseMapping.csv]: No notice file

**Comment by jsvd** [comment]
ugh well spotted @robbavey , I messed it up somewhere along the way

File: tools/dependencies-report/src/main/resources/licenseMapping.csv (hunk 5 above)
Comment (jsvd) [reply tools/dependencies-report/src/main/resources/licenseMapping.csv]: @andrewvc I don't see this being included at all in logstash or jruby (despite being in jruby's copying file), should we keep this?

**Comment by jsvd** [comment]
merged in https://github.com/elastic/logstash/pull/9886

**Pull request closed without merging**